src/Dfc.CourseDirectory.Core/ReferenceData/Ukrlp/UkrlpSyncHelper.cs

```diff
@@ -54,25 +57,32 @@ public async Task SyncAllProviderData(DateTime updatedSince)
                 {
                     updatedCount++;
                 }
+
+                if ((createdCount + updatedCount) % 200 == 0)
```

Comment:
???

Reply:
This spits out an info level log every 200 items

```
[24/09/2020 12:48:35] UKRLP Sync: processed provider 200 of 5622, UKPRN: 10001725...  << Info level (will show in azure logs by default)
```

Reply:
Buy why?

Reply:
Because the function-app takes quite a while to run. This output means the there's evidence of progress in the logs instead of just silence till it's finished.